cmd/start.go

```diff
@@ -20,9 +22,10 @@ import (
 var StartCommand cli.Command
 
 const (
-	bootstrapFlag = "bootstrap"
-	portFlag      = "port"
-	portShort     = "p"
+	bootstrapFlag    = "bootstrap"
+	portFlag         = "port"
+	portShort        = "p"
+	stakeTimeoutFlag = "stake-timeout"
```

Comment:
Would love to align the naming here to the excellent name you have for the function itself: `--wait-for-stake`. That also gives us a perfect short flag that won't conflict with any common expectations I think, `-w`.

Going to trigger the build here to see if there's anything lurking.

Reply:
I agree. I have made this change